src/resolve/resolved-dns-query.c

```diff
@@ -515,6 +516,7 @@ int dns_query_new(
                 .answer_dnssec_result = _DNSSEC_RESULT_INVALID,
                 .answer_protocol = _DNS_PROTOCOL_INVALID,
                 .answer_family = AF_UNSPEC,
+                .request_name = NULL
```

Comment:
We generally don't initialize fields to NULL, this line can be skipped.

Reply:
Removed in latest patch